python/paddle/tensor/math.py

```diff
@@ -2646,19 +2646,32 @@ def lerp(x, y, weight, name=None):
     if in_dygraph_mode():
         check_type(weight, 'weight', (float, paddle.Tensor, Variable), 'lerp')
         if isinstance(weight, float):
-            weight = paddle.to_tensor(weight, dtype=x.dtype, place=paddle.CPUPlace())
+            weight = paddle.to_tensor(weight, dtype=x.dtype)
         return _C_ops.lerp(x, y, weight)
 
     check_variable_and_dtype(x, 'x', ['float32', 'float64'], 'lerp')
     check_variable_and_dtype(y, 'y', ['float32', 'float64'], 'lerp')
-    check_variable_and_dtype(weight, 'Weight', ['float32', 'float64'], 'lerp')
+    check_variable_and_dtype(weight, 'weight', ['float32', 'float64'], 'lerp')
 
     helper = LayerHelper('lerp', **locals())
     inputs = {'X': x, 'Y': y, 'Weight': weight}
     out = helper.create_variable_for_type_inference(dtype=x.dtype)
     helper.append_op(type='lerp', inputs=inputs, outputs={'Out': out})
     return out
 
+@inplace_apis_in_dygraph_only
+def lerp_(x, y, weight, name=None):
```

Comment:
Please also add a simple test to this API. Any code without test is unsafe ^_^

Reply:
![image](https://user-images.githubusercontent.com/23427135/144822806-6e85cfd6-354d-469d-927a-09e6da72ea1b.png)